components/Currency/CurrencyIcon/CurrencyIcon.tsx

```diff
@@ -30,7 +32,23 @@ export const SNXIcon =
 export const getSynthIcon = (currencyKey: CurrencyKey) =>
 	`https://raw.githubusercontent.com/Synthetixio/synthetix-assets/master/synths/${currencyKey}.svg`;
 
-export const CurrencyIcon: FC<CurrencyIconProps> = ({ currencyKey, type, ...rest }) => {
+export const CurrencyIconContainer: FC<CurrencyIconProps> = (props) => (
```

Comment:
```({isDeprecated})```

Reply:
```
{isDeprecated && <DeprecatedXIconContainer>
				<Svg src={DeprecatedXIcon} />
			</DeprecatedXIconContainer>}
```

Reply:
the `props` are being based in the next line